ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1049,13 +1058,25 @@ def input(
         # always check if file is present as the grpc and MAPDL errors
         # are unclear
         if self._local:
-            if not os.path.isfile(fname):
-                raise FileNotFoundError('Unable to locate filename "%s"' % fname)
-
-            if not os.path.dirname(fname):
-                filename = os.path.join(os.getcwd(), fname)
+            if os.path.isdir(fname):
+                raise IOError('It seems you give a directory path, not a file path or file name.')
```

Comment:
ValueError seems more correct here.
```suggestion
                raise ValueError('It seems you give a directory path, not a file path or file name.')
```